recipes/spruce/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+go_compiler:
```

Comment:
I would prefer that you use the compiler directly below and not have a cbc.yaml.

Reply:
Perfect, thanks!